lib/src/interfaces/Options.ts

```diff
@@ -1089,6 +1094,27 @@ export interface OverlayOptions {
    * Set this to true if your Overlay contains a TextInput.
    */
   handleKeyboardEvents?: boolean;
+
+  /**
+   * Attach overlay to anchor view in a certain layer of layout as a tooltip
+   */
+  attach?: {
+    /**
+     * layout id to look for to add as a layer
+     * which can be componentId or stackId or bottomTabsId.
+     */
+    layoutId: string;
+    anchor: {
```

Comment:
this one should be optional, for when the user wants to attach an overlay to a component with no anchor

Reply:
did not we talk about it to be in the next phase? making it optional will not show anything